media/public/include/IWebAudioPlayer.h

```diff
@@ -35,6 +35,11 @@
 #include <stdint.h>
 #include <string>
 
+namespace firebolt::rialto::client
```

Comment:
fwd declarations

Reply:
Added comment in the code to explain why I think this forward declaration might be necessary